api/views.py

```diff
@@ -50,11 +50,14 @@ class SearchParams:
     count: int = None
     start: Optional[int] = None
     order_by: Optional[str] = None
-    sort: str = "asc"
+    sort: str = "desc"
 
     @staticmethod
     def from_request(
-        request: HttpRequest, max_count: int, allowed_sort_columns: List["str"]
+        request: HttpRequest,
+        max_count: int,
+        allowed_order_by_columns: List["str"],
```

Comment:
Can we get rid of `"` in `"str"`?

Reply:
Yes, thanks! I used literals there before, so it's a leftover from that, probably. Fixed.